### PR TITLE
Switch to OneTrust Cookie Consent

### PR DESCRIPTION
**Pull request by jacksoncvm**

### CODE REVIEW

**Comment by jacksoncvm** [comment]
@simonbasle Please review

File: src/main/resources/static/templates/layout.html [hunk 1]
```diff
@@ -16,13 +16,18 @@
...
-            new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-        j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-        'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-    })(window,document,'script','dataLayer','GTM-N4T4SNK');</script>
-    <!-- End Google Tag Manager -->
+    <!-- OneTrust Cookie Consent -->
+    <meta content='60a709d9-50a1-442d-8f59-072e08a3ddb4' name='onetrust-data-domain'>
+    <meta content='https://tags.tiqcdn.com/utag/vmware/microsites-privacy/prod/utag.js' name='microsites-utag'>
+    <script src="https://code.jquery.com/jquery-2.1.4.js"></script>
+    <script src='//www.vmware.com/files/templates/inc/utag_data.js'></script>
```
Comment (simonbasle) [inline src/main/resources/static/templates/layout.html:23]: missing `https:` before the `//` here and on line 23?

File: src/main/resources/static/templates/layout.html [hunk 2]
```diff
@@ -154,12 +156,13 @@ <h1 id="logo"><a href="/"><strong>Project Reactor</strong></a></h1>
...
+                    © 2021 <a href="https://www.vmware.com/" class="pivotal">VMware, Inc.</a> or its affiliates. All rights reserved | <a
                         href="https://www.vmware.com/help/legal.html">Terms
                     of Use</a> |
-                    <a href="https://www.vmware.com/help/privacy.html">Privacy</a> |
-                    <a href="/security-policy">Security Policy</a> |
-                    <span id="teconsent"></span>
+                    <a href="https://www.vmware.com/help/privacy.html">Privacy</a> | 
+                    <a href="/security-policy">Security Policy</a> | 
+                    <a href="https://www.vmware.com/help/privacy/california-privacy-rights.html">Your California Privacy Rights</a> | 
+                    <a class="ot-sdk-show-settings">Cookie Settings</a>
```
Comment (simonbasle) [inline src/main/resources/static/templates/layout.html:165]: this doesn't seem to do anything, when running the site locally, even with the above `https:` modification (which would supposedly bring in the OT sdk) 🤔

File: src/main/resources/static/templates/layout.html (hunk 1 above)
Comment (simonbasle) [reply src/main/resources/static/templates/layout.html:23]: also, these scripts seem like they might be heavily `vmware.com` oriented. can you confirm they're supposed to work on other domains?

**Review by simonbasle — CHANGES_REQUESTED** [review]

**Comment by simonbasle** [comment]
thanks for the PR @jacksoncvm. I added a couple of comments/questions.

**Comment by simonbasle** [comment]
I was able to test it on a temporary deployment, and it turns out I had some adblocker preventing the cookie consent settings to pop up.

**Pull request merged**